src/IconButton.js

```diff
@@ -28,8 +42,8 @@ class IconButton extends React.Component<IconButtonProps, {}> {
                 className={classes}
```

Comment:
Screen Readers don't know that the IconButton is disabled and we need to inform that the button is disabled by using aria-disabled property. 
```
aria-disabled={this.props.disabled}
```

Reply:
Done.